packages/webapp/src/_app/hooks/queries.ts

```diff
@@ -354,3 +413,39 @@ export const useEncryptedData = (scope: EncryptionScope, id: string) =>
         ...queryEncryptedData(scope, id),
         enabled: Boolean(id),
     });
+export const useOngoingElectionData = (
+    queryOptions: any = {}
+): UseQueryResult<Election | undefined> => {
+    const { data: loggedInMember } = useCurrentMember();
+    // GET highestRandIndexParticipatedIn
+    const { data: memberStats } = useMemberStats();
+    const { data: electionState } = useCurrentElection();
+    const { data: myDelegation } = useMyDelegation();
+
+    // GET participants for ongoing round
+    const { data: membersInOngoingRound } = useMemberGroupParticipants(
+        loggedInMember?.account,
+        memberStats?.ranks?.length
+    );
+    console.info("useOED().membersInOngoingRound:", membersInOngoingRound);
+    let { data: votingMemberData } = useMemberDataFromVoteData(
+        membersInOngoingRound
+    );
+    console.info("useOED().votingMemberData:", votingMemberData);
+
+    const { queryKey, queryFn } = queryOngoingElectionData(
+        memberStats,
+        votingMemberData,
+        electionState,
+        myDelegation
+    );
+    return useQuery<Election, Error>({
+        queryKey,
+        queryFn,
+        enabled:
+            Boolean(loggedInMember) &&
+            Boolean(memberStats) &&
+            Boolean(electionState) &&
+            Boolean(myDelegation),
+    });
```

Comment:
I think we could shorten this to `Boolean(loggedInMember && memberStats && electionState && myDelegation),`.

Do we need the check for `loggedInMember` and should we add a check for the presence of `votingMemberData` instead?